README.md

```diff
@@ -27,31 +15,27 @@ software systems.  The key features it provides are:
 * Powerful abstractions
 * Declarative concurrency
 
-Let's go through some practical examples to see it working. You
-can find the working code of these examples in the [streamly-examples
+This guide introduces you to streamly using some
+practical examples to see it working. You can find the
+working code of these examples in the [streamly-examples
```

Comment:
Will this link be rendered properly?

Reply:
Yes. See https://github.com/composewell/streamly/tree/readme .